tools/code-owners-parser/ci.yml

```diff
@@ -0,0 +1,28 @@
+# NOTE: Please refer to https://aka.ms/azsdk/engsys/ci-yaml before editing this file.
```

Comment:
Do we need a ci for this library, as I assume we aren't going to publish it? It might be better to just build it as part of the other consuming library CI pipelines.

Reply:
OK. I wouldn't publish that until we figure out if we have any consumer for it though.